### PR TITLE
fix(svg-to-jsx): Attribute in camelCase

### PR DESCRIPTION
**Pull request by NiniGeek**
## What it does

Switch svg attribute name to kebabCase

## Linear ticket
Resolves DEV-1088

### CODE REVIEW

File: parsers/svg-to-jsx/svg-to-jsx.spec.ts [hunk 1]
```diff
@@ -72,4 +72,21 @@ describe('Svg to jsx', () => {
...
+    expect(result[0].value.content).toMatch(/strokeLinejoin/)
+    return;
+  });
+  it('Validate kebabCase on attribute with snapshot', async () => {
+    const tokens = seeds().tokens.filter(({ name }) => name === 'activity');
+    const result = await svgToJsx(tokens as InputDataType, undefined, libs as LibsType);
+    if (result instanceof Error) return fail(result);
+    expect(result[0].value.content).toMatchSnapshot();
+    return;
+  });
```
Comment (NiniGeek) [inline parsers/svg-to-jsx/svg-to-jsx.spec.ts:91]: What do you prefer?

File: parsers/svg-to-jsx/svg-to-jsx.parser.ts [hunk 2]
```diff
@@ -73,16 +75,52 @@ export default async function (
...
+                let value = element._;
+                if (!value && element.$$) {
+                  value = Object.assign({}, ...element.$$ )
+                }
+                if (element.$) {
+                  value = { ...value, ...element.$ }
+                }
+                return {
+                  [tag]: value,
+                };
```
Comment (NiniGeek) [inline parsers/svg-to-jsx/svg-to-jsx.parser.ts:105]: Did I miss a more suitable javascript function?

File: parsers/svg-to-jsx/svg-to-jsx.spec.ts (hunk 1 above)
Comment (AMoreaux) [reply parsers/svg-to-jsx/svg-to-jsx.spec.ts:91]: I prefer to avoid the usage of snapshots because it's too macro.

**Review by AMoreaux — CHANGES_REQUESTED** [review]

File: tsconfig.json [hunk 3]
```diff
@@ -10,6 +10,9 @@
     "inlineSourceMap": false,
     "inlineSources": false,
     "resolveJsonModule": true,
-    "useUnknownInCatchVariables": false
+    "useUnknownInCatchVariables": false,
+    "lib": [
+      "dom"
+    ],
```
Comment (NiniGeek) [inline tsconfig.json:16]: Test failed because of this. What I can add "esnext" or "es2019" or more ?

Comment (AMoreaux) [reply tsconfig.json:16]: You can add `es2020`

**Review by AMoreaux — APPROVED** [review]

**Review by AMoreaux — APPROVED** [review]

**Review by AMoreaux — APPROVED** [review]

**Pull request merged**